.rubocop.yml

```diff
@@ -2,6 +2,12 @@ AllCops:
   TargetRubyVersion: 3.0
   DisabledByDefault: true
   SuggestExtensions: false
+  Exclude:
```

Comment:
How about adding a comment why we exclude them?

Reply:
Good point. Added some comments.